otter/tap/api.py

```diff
@@ -289,11 +289,12 @@ def on_client_ready(_):
             # Give dispatcher to Otter REST object
             otter.dispatcher = dispatcher
 
-            # Set the client after starting
+            # Set the client and dispatcher after starting
             # NOTE: There is small amount of time when the start is
             # not finished and the kz_client is not set in which case
             # policy execution and group delete will fail
             store.kz_client = kz_client
+            store.dispatcher = dispatcher
```

Comment:
Obviously it would be nicer if this were a discrete event on `Otter` (use automat!), or even better constructor arguments.  Clearly this has been previously considered though, given the comment.